CMSgov/design-system
# [NO-TICKET] Typography documentation updates

## Summary

- Updated the heading typography page content 
- Updated the body typography page content

## How to test

1. Run the doc site `yarn start` and view the Headings and body pages
2. Run `yarn storybook` to see that the typography stories for responsive headings have been updated

## Checklist




### If this is a change to code:


### If this is a change to documentation:
